src/tlo/scenario.py

```diff
@@ -327,7 +326,7 @@ def get_sample(self, draw, sample_number):
         # Instead of using the random number generator to create a seed for the simulation, we use an integer hash
         # function to create an integer based on the sum of the draw_number and sample_number. This means the
```

Comment:
I this description still correct (and/or was it correct for previous implementation)? It looks like the only `sample_number` feeds in to generated seed not `draw_number`.